.github/workflows/ci.yml

```diff
@@ -17,10 +17,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
-        python-version: [3.8, 3.9, '3.10']
-        exclude:
-          - os: windows-latest
-            python-version: 3.8
+        python-version: [3.9, '3.10', 3.11]
```

Comment:
@oyamad does the `3.11` require quotes like `3.10`?

Reply:
I believe only `3.10` requires quotes.